### PR TITLE
Bunch of quickfixes

### LINKED ISSUE #390: Fields in Task display should be clickable links
The Task page shows learner, event, and task - the first two should be clickable links.

### LINKED ISSUE #403: "View Task" should have a "delete me" button
The page displaying a single task should have a "delete this task" button - we shouldn't have to go to the "Event" page to delete the task.

### LINKED ISSUE #404: Eventbrite key on New Event page should be labelled "Eventbrite key"
Currently "Reg key", which is confusing and inconsistent.

### LINKED ISSUE #405: Link to Eventbrite would be nice
It would be great to have a link to the Eventbrite "Manage event" page for an event if there's an Eventbrite key. The URL is https://www.eventbrite.com/myevent?eid=nnnnnnnnnnn

### LINKED ISSUE #407: Link to "New Airport" page from "New Person" page?
It would be handy to have a link to the "New Airport" page near the Airport field on the New Person page, since you often need to add a new airport once you realize the one you need is not there.

### LINKED ISSUE #408: Better field names on New Airport page
Can we change "Iata" to "IATA Code" 

Add a link to https://www.world-airport-codes.com/ or something similar.

Change "Fullname" to "Airport Name"

### LINKED ISSUE #409: Link from Person directly to Event, not Task
At the bottom of the Person page is a list of tasks -- they're links which lead to the Task page. Would be much more useful if those links were directly to the Event. (There's no link to the Event on the Task page, either -- you have to copy the event name and paste it in the search field to eventually get to the Event page.)

In fact, I can't think of any reason I would need to see a Task page -- tasks are only relevant in the context of the Event or the Person, not as standalone items.

### LINKED ISSUE #410: In New Event page, End date should auto-fill
Once the user has filled in the Start date, the End date should autofill as the next day.

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
More details in commit messages.

Fixes #390.
Fixes #403.
Fixes #404.
Fixes #405.
Fixes #407.
Fixes #408.
Fixes #409.
Fixes #410.

### CODE REVIEW

File: workshops/forms.py [hunk 1]
```diff
@@ -211,6 +211,9 @@ class Meta:
                   'tags', 'url', 'reg_key', 'admin_fee', 'invoiced',
                   'attendance', 'notes')
 
+    class Media:
+        js = ('calendar_popup.js', )
```
Comment (gvwilson) [inline workshops/forms.py]: I have no idea what this does :-)

File: workshops/models.py [hunk 2]
```diff
@@ -39,8 +39,9 @@ def get_absolute_url(self):
 class Airport(models.Model):
     '''Represent an airport (used to locate instructors).'''
 
-    iata      = models.CharField(max_length=STR_SHORT, unique=True)
-    fullname  = models.CharField(max_length=STR_LONG, unique=True)
+    iata = models.CharField(max_length=STR_SHORT, unique=True, verbose_name="IATA code",
+                            help_text='You can look it up here: <a href="https://www.world-airport-codes.com/">https://www.world-airport-codes.com/</a>')
```
Comment (gvwilson) [inline workshops/models.py]: Change help to `<a href="...">Look up code</a>`.

**Comment by gvwilson** [comment]
One small change to help, then :+1: to merge.

**Comment by pbanaszkiewicz** [comment]
Rebased against master, added comment to the `class Media`, shortened `iata.help_text`, added a merge migration.

**Pull request merged**